packages/react/src/components/DataTable/TableToolbarSearch.js

```diff
@@ -51,6 +51,13 @@ const TableToolbarSearch = ({
     }
   }, [controlled, expandedState]);
 
+  useEffect(() => {
+    if (defaultValue) {
+      setExpandedState(true);
+      onChangeProp('', defaultValue);
+    }
+  }, [defaultValue, onChangeProp]);
```

Comment:
Just want to double-check; Wondering if we want to react to change in `defaultValue`. The thought came up because `defaultValue` in `<input>` is effective only in creation time.

Reply:
I don't think it's a problem since it won't change. If I remove it I get a linting error:

<img width="782" alt="Screen Shot 2020-01-17 at 4 06 44 PM" src="https://user-images.githubusercontent.com/11928039/72654222-7793fa00-3943-11ea-9fe8-6f2148c79bde.png">

Reply:
Thank you for your thoughts - Lint error is something we can ignore as I believe. Wondering what you meant by "it won't change" - Does it translate to "our users _should not_ change `defaultValue` after creation"? The thought came because it's a React prop and thus component users can change it technically.

Reply:
Thank you for the update @tw15egan! (Seems that I signed this off already)